it/uk/gov/hmrc/agentmapping/MappingControllerISpec.scala

```diff
@@ -9,490 +9,212 @@ import play.api.libs.ws.WSClient
-      givenIndividualRegistrationExists(utr)
-      createMappingRequest(saOnlyIdentifier).putEmpty().status shouldBe 201
-      createMappingRequest(saAndVatIdentifiers).putEmpty().status shouldBe 201
-
-      verifyAuditRequestSent(
-        1,
-        event = AgentMappingEvent.KnownFactsCheck,
-        detail = Map(
-          "knownFactsMatched" -> "true",
-          "utr" -> "2000000000",
-          "agentReferenceNumber" -> "AARN0000002",
-          "authProviderId" -> "testCredId"),
-        tags = Map(
-          "transactionName" -> "known-facts-check",
-          "path" -> "/agent-mapping/mappings/2000000000/AARN0000002/IRAgentReference~A1111A"))
-
-      verifyAuditRequestSent(
-        1,
-        event = AgentMappingEvent.KnownFactsCheck,
-        detail = Map(
-          "knownFactsMatched" -> "true",
-          "utr" -> "2000000000",
-          "agentReferenceNumber" -> "AARN0000002",
-          "authProviderId" -> "testCredId"),
-        tags = Map(
-          "transactionName" -> "known-facts-check",
-          "path" -> "/agent-mapping/mappings/2000000000/AARN0000002/IRAgentReference~A1111A~AgentRefNo~101747696"))
-
-      verifyAuditRequestSent(
-        1,
-        event = AgentMappingEvent.CreateMapping,
-        detail = Map(
-          "saAgentRef" -> "A1111A",
-          "agentReferenceNumber" -> "AARN0000002",
-          "authProviderId" -> "testCredId",
-          "duplicate" -> "false"),
-        tags = Map(
-          "transactionName" -> "create-mapping",
-          "path" -> "/agent-mapping/mappings/2000000000/AARN0000002/IRAgentReference~A1111A"))
-
-      verifyAuditRequestSent(
-        1,
-        event = AgentMappingEvent.CreateMapping,
-        detail = Map(
-          "saAgentRef" -> "A1111A",
-          "agentReferenceNumber" -> "AARN0000002",
-          "authProviderId" -> "testCredId",
-          "duplicate" -> "true"),
-        tags = Map(
-          "transactionName" -> "create-mapping",
-          "path" -> "/agent-mapping/mappings/2000000000/AARN0000002/IRAgentReference~A1111A~AgentRefNo~101747696"))
-
-      verifyAuditRequestSent(
-        1,
-        event = AgentMappingEvent.CreateMapping,
-        detail = Map(
-          "vatAgentRef" -> "101747696",
-          "agentReferenceNumber" -> "AARN0000002",
-          "authProviderId" -> "testCredId",
-          "duplicate" -> "false"),
-        tags = Map(
-          "transactionName" -> "create-mapping",
-          "path" -> "/agent-mapping/mappings/2000000000/AARN0000002/IRAgentReference~A1111A~AgentRefNo~101747696"))
-
+    // Then test all fixtures at once
+    s"capture all ${fixtures.size} enrolments" when {
+      s"return created upon success" in {
+        givenUserIsAuthorisedForMultiple(fixtures)
+        givenIndividualRegistrationExists(utr)
+        createMappingRequest().putEmpty().status shouldBe 201
```

Comment:
Is it worth verifying the expected audit events were also sent?

Reply:
Actually, no worries, I can see this is done in the `splitFixtures` tests below.